evm/contracts/examples/ConversionRate.sol

```diff
@@ -152,6 +152,23 @@ contract ConversionRate is ChainlinkClient, Ownable {
     authorizedRequesters[_requester] = _allowed;
   }
 
+  function cancelRequest(
+    bytes32 _requestId,
+    uint256 _payment,
+    uint256 _expiration
+  )
+    public
+    ensureAuthorizedRequester()
+  {
+    cancelChainlinkRequest(
+      _requestId,
+      _payment,
+      this.chainlinkCallback.selector,
+      _expiration
+    );
+    delete requestAnswers[_requestId];
```

Comment:
Just bringing this up here, should we also clean up the `answers` mapping in case another node has responded? This is primarily thinking if we received less than the min responses and want to cancel multiple requests, we'd want to clean up the remaining stored answers that haven't ran through aggregation.

Reply:
Good call.